mage_ai/orchestration/triggers/api.py

```diff
@@ -28,6 +30,13 @@ def create_and_start_pipeline_run(
         payload,
     )
 
+    # TODO: make sure git syncs are not run concurrently if there are
```

Comment:
This could actually be an issue. Let's not run git sync in the API call.

Reply:
ok i'll remove it